tests/scripts/run_performance.sh

```diff
@@ -120,6 +129,8 @@ run_device_perf_models() {
         env WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/metal_BERT_large_11/tests -m $test_marker
 
         env WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/falcon7b_common/tests -m $test_marker
+
+        # env WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/wormhole/mnist/tests -m $test_marker
```

Comment:
no device perf test?

Reply:
As I'm encountering errors related to the CIs, I’ve moved the test to line 104 to verify if the device performance test runs. This is an experiment to help identify what might be causing the issue.